.github/CODEOWNERS

```diff
@@ -37,7 +37,7 @@
 #/avm/res/databricks/access-connector/ @Azure/avm-res-databricks-accessconnector-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/databricks/workspace/ @Azure/avm-res-databricks-workspace-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/data-factory/factory/ @Azure/avm-res-datafactory-factory-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/data-protection/backup-vault/ @Azure/avm-res-dataprotection-backupvault-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/data-protection/backup-vault/ @Azure/avm-res-dataprotection-backupvault-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Team seems to be missing

Reply:
And added to parent team as per https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
Exist already:
@Azure/avm-res-consumption-budget-module-owners-bicep
@Azure/avm-res-consumption-budget-module-contributors-bicep

Reply:
May still be missing the owner assignment / its approval

Reply:
both teams exist and parent teams are assigned

Reply:
True true